config/dev.exs

```diff
@@ -65,3 +65,5 @@ config :phoenix, :stacktrace_depth, 20
 
 # Initialize plugs at runtime for faster development compilation
 config :phoenix, :plug_init_mode, :runtime
+
+config :tic_tac_toe_api, seed: ""
```

Comment:
What does a seed of `""` do?

Reply:
we wanted to make it so that in a prod/dev environment, there is no seed set to make the PRNG at constant seed. Is it better to leave it blank (if elixir is okay with that)?

Reply:
What you did works.